compiler/src/test/java/com/squareup/anvil/compiler/codegen/BindingModuleGeneratorTest.kt

```diff
@@ -1,10 +1,10 @@
 package com.squareup.anvil.compiler.codegen
 
 import com.google.common.truth.Truth.assertThat
-import com.sqareup.anvil.compiler.internal.testing.AnyDaggerComponent
-import com.sqareup.anvil.compiler.internal.testing.anyDaggerComponent
-import com.sqareup.anvil.compiler.internal.testing.daggerModule
-import com.sqareup.anvil.compiler.internal.testing.isAbstract
+import com.squareup.anvil.compiler.internal.testing.AnyDaggerComponent
+import com.squareup.anvil.compiler.internal.testing.anyDaggerComponent
+import com.squareup.anvil.compiler.internal.testing.daggerModule
+import com.squareup.anvil.compiler.internal.testing.isAbstract
 import com.squareup.anvil.annotations.MergeComponent
```

Comment:
Well, you fixed them, but now your imports are all out of order 😆

Reply:
Gosh, I swear I thought of that first and then forgot 😭